requirements.txt

```diff
@@ -15,4 +15,5 @@ typing_extensions==4.0.1
 Werkzeug==2.0.2
 zipp==3.8.0
 nox==2022.1.7
-psutil==5.9.1 
\ No newline at end of file
+psutil==5.9.1
+packaging==21.3
```

Comment:
This package was already installed (probably a dependency of a dependency). I added it here because now we use it in the component-tests.